client/python/examples/README.md

```diff
@@ -0,0 +1,246 @@
+# Python Client Examples
+
+Currently there are five examples:
+
+- [Intro to using the Python Client](#simplepy)
```

Comment:
Since you still call these "simple.py" and similar below, consider adding those names here
e.g.
```suggestion
- [Intro to using the Python Client - simple.py](#simplepy)
```